plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -84,7 +84,7 @@ struct Database {
     }
 
     bool hasLostMarkings(Path path) @trusted {
-        auto s = format!"SELECT COUNT(*) FROM %s WHERE path=:path LIMIT 1"(markedMutantTable);
+        immutable s = format!"SELECT COUNT(*) FROM %s WHERE path=:path LIMIT 1"(markedMutantTable);
```

Comment:
:100:

Reply:
I do not understand how this query finds those that have lost a marking. Could you enlighten me?

Reply:
No, that is completely true. I started out making the query more complex, and then slimmed it down to current implementation. Did not change the name that is.

I guess you could say that it WAS how i used it. I will correct it tomorrow.